docs/room-acoustics.md

```
@@ -533,7 +537,407 @@ values (thirds `L'nT,w = 79`, `CI = −11`; octave `54`, `CI = 0`).
+corrected = background_correction([30.0, 33.0, 50.0], [24.0, 32.0, 30.0])
+print(np.round(corrected, 1))                 # [28.7 31.7 50.0]  (1.3 dB cap twice)
+
+lab.rating.plot()   # measured R vs shifted ISO 717-1 reference (needs matplotlib)
+```
+
+A margin at or below 6 dB emits a `LabInsulationWarning` and flags the band as
+the limit of measurement; catch it with `warnings.simplefilter("error",
+LabInsulationWarning)`. The automatic rating is formed only when exactly 16
+one-third-octave or 5 octave values are supplied (`rating` is `None` otherwise).
+
+#### `lab_airborne_insulation()` / `lab_impact_insulation()` parameters
```

markdownlint-cli2 (0.22.1)

[warning] Heading levels should only increment by one level at a time
Expected: h3; Actual: h4

(MD001, heading-increment)